### DISCUSSION TITLE
[Question]: GPT official website has a new interface. Does librechat plan to change the interface?

### DISCUSSION THREAD

**Discussion by 13531446481** [danny-avila/LibreChat#3077] [Q&A]
### What is your question?

GPT official website has a new interface. Does librechat plan to change the interface?

### More Details

GPT official website has a new interface. Does librechat plan to change the interface?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Thanks for your question. Though there are some open PR's, It's not a priority, as this takes development time away from new features, and I would like to iterate on features that are not offered by ChatGPT.
